src/util.js

```diff
@@ -9,3 +9,14 @@ export function assign(obj, props) {
 	for (let i in props) obj[i] = props[i];
 	return /** @type {O & P} */ (obj);
 }
+
+/**
+ * Remove a child node from its parent if attached. This is a workaround for
+ * IE11 which doesn't support `Element.prototype.remove()`. Using this function
+ * is smaller than including a dedicated polyfill.
+ * @param {Node} node The node to remove
+ */
+export function removeNode(node) {
```

Comment:
This is an opportunity with a module build to remove the additional method via build system transform.

Reply:
We're all rooting for your proposal :+1: :+1: